lib/features/evolution_tab/widgets/evolution_tile.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/material.dart';
+import 'package:pokedex/utilities/constants.dart';
+import 'package:pokedex/utilities/extensions.dart';
+
+class EvolutionTile extends StatelessWidget {
+  const EvolutionTile({
+    this.name,
+    this.id,
+    this.isNotFirstEvolution,
+  });
+
+  final String? name;
+  final int? id;
+  final bool? isNotFirstEvolution;
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      children: [
+        if (isNotFirstEvolution ?? false)
+          (Padding(
+            padding: EdgeInsets.all(2),
+            child: Icon(
```

Comment:
Is this the same as `Icon(Icons.arrow_forward_ios_rounded)`?

Reply:
ah yes same